.golangci.yml

```diff
@@ -64,6 +63,8 @@ linters:
         - XXX
     govet:
       enable-all: true
+      disable:
+        - fieldalignment
```

Comment:
Are you planning to (re)enable this in another PR? (This comment applies to all NFs where the `fieldalignment` was disabled)